paper/paper.md

```diff
@@ -27,6 +27,25 @@ bibliography: paper.bib
 * introduction to package
 * maybe quick example
 
+section 1:
+Intro, statement of need
+
+HELP NEEDED: idea collection for statement of need
+* Paragraph1: focus on need for spatial analysis?
+
+* Paragraph2: focus on need to visualize spatial analysis?
+
```

Comment:
could borrow from [here](https://github.com/sjsrey/gds_scipy16/blob/master/content/part1/02_geovisualization.ipynb)

Reply:
I think that's exactly it. To reiterate, some of the ideas I'd include in the statement of need:

- There's a lot of visualisation toolkits (in Python), less of geovisualisation, and non (until splot) for visualisation of spatial analytics.
- The need here is to create bespoke "views" of the outputs from the statistics implemented in PySAL (and beyond, e.g. VBA mapping!) and, more generally, of more sophisticated statistical visualisation.
- Because of the two points above, splot does *not* conflict with existing packages in the python eco-system, or reinvent any wheels. Indeed, although most of the current development is in `matplotlib`, we've always had in mind being framework-independent. In that sense, `splot` is more like a "grammar" of views we think are important and useful in spatial analysis/geographic data science, with a `matplotlib` implementation (currently).

Reply:
One more thought, I think the motivation in Serge's link to the notebook that "people kept asking for it" is (factually) correct but, for the paper, I'd rephrase it to say something like: 

> "over time, in parallel to the rise of data science, the ecosystem of tools to store, manage and analyse spatial data evolved in ways that gave more relevance to integrated code-based environments such as  Jupyter; and less to disconnected, one-purpose point-and-click tools such as traditional desktop GIS packages. In this context, PySAL could be integrated with "